source/manual/pact-testing.html.md

```diff
@@ -61,21 +61,17 @@ _If the app already had some Pact tests, follow [the steps for changing existing
   - [Consumer example](https://github.com/alphagov/gds-api-adapters/pull/1066) (Note: since this example was written [we now store pact tests in the `tests/pacts` directory](https://github.com/alphagov/gds-api-adapters/blob/main/test/pacts)).
   - [Provider example](https://github.com/alphagov/imminence/pull/644).
 
-1. Check the consumer tests pass locally.
-  - CI won't do this for you yet because it doesn't know about the new app.
+1. Check the tests pass locally for both provider and consumer
+  - CI won't be able to test them yet as they won't be pushed to the pact broker.
 
 1. Merge the consumer tests for the new app.
-  - The build will pass because it runs with the old Jenkinsfile.
+  - This will cause the pact tests to be published to the pact broker, given the [project has been configured to do so](https://github.com/alphagov/gds-api-adapters/blob/bcc8e58eccf69dd37657d13156cbe11c07535844/.github/workflows/ci.yml#L37-L51).
```

Comment:
Seems unnecessarily wordy - how about:

```suggestion
  - This will cause the [pact tests to be published to the pact broker](https://github.com/alphagov/gds-api-adapters/blob/bcc8e58eccf69dd37657d13156cbe11c07535844/.github/workflows/ci.yml#L37-L51).
```